src/Log.ts

```diff
@@ -97,9 +101,15 @@ export class Log {
 
             // if not false and conditions pass, log it
             return null;
-        });;
-    }
+        });
 
+        setTimeout(() => {
+            if (this.printLog) {
+                Log.deferred.forEach(x => x(this))
+            }
+            Log.deferred.length = 0
```

Comment:
Isn't it better to reset the array?

` Log.deferred = []`

Reply:
It has the same effect, I can change it: changed